src/components/artwork/metadata.tsx

```diff
@@ -9,14 +9,19 @@ import ArtworkDetails from "./details"
 
 export interface ArtworkMetadataProps extends React.HTMLProps<ArtworkMetadata> {
   artwork: any
+  extended?: boolean
 }
 
 export class ArtworkMetadata extends React.Component<ArtworkMetadataProps, null> {
+  static defaultProps = {
+    extended: false,
```

Comment:
I'm coming to this way late, but I'm taking a look at the artwork bricks, and this seems to be an issue.

Namely, this default means this component won't show the sale line or the `ArtworkContact` component, which is generally always a part of artwork bricks. It might even be _the_ most important part! You'd need to pass in 

The `Artwork/GridItem` and `Artwork` index components don't work with this- neither allows passing in the`extended` prop down to this component.

Sure enough, the Gene page artwork bricks aren't correct:

<img width="310" alt="screen shot 2017-10-26 at 11 23 42 am" src="https://user-images.githubusercontent.com/1457859/32061658-37f43696-ba40-11e7-87ed-a35c3027a209.png">

There should be a price and contact button underneath, like:

<img width="171" alt="screen shot 2017-10-26 at 11 24 09 am" src="https://user-images.githubusercontent.com/1457859/32061665-3d0cb6bc-ba40-11e7-81e8-fea381663dd4.png">

Do you recall if this was just fallout? Most likely all the base components should allow this to be passed thru (with a default of true), and then the Loyalty component can override this to false, if it needs to.

Reply:
The default should definitely be `true` indeed. Also, do we need not extended components anywhere besides these loyalty program bricks? Otherwise maybe it can just be a similar but separate component that doesn’t include the metadata component?

Reply:
Looking at https://github.com/artsy/reaction/blob/a486ed3651ee5e0a4ae5852c05d1ed3f6dd5cb6f/src/components/artwork/index.tsx I think even _more_ that separating bricks for loyalty and all others is the way to go. Normal bricks don’t need an overlay either, do they?